slices/dash.yaml

```diff
@@ -0,0 +1,11 @@
+package: dash
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - base-passwd_data
+      - base-files_base
+    contents:
+      /bin/dash:
+      /bin/sh:
```

Comment:
Similar to my comment in https://github.com/canonical/chisel-releases/pull/61#discussion_r1326873419, we should probably drop this symlink from the slice definition file and add it manually afterwards, until chisel supports conflicting paths.

Reply:
well, IMO that makes sense in https://github.com/canonical/chisel-releases/pull/61#discussion_r1326873419 cause in that case the file is added via a maintainer script. In this case, the file is an integral part of the deb, and thus I believe it is safe to have it here